camviewer.ui

```diff
@@ -625,6 +663,81 @@ Vmin\</string>
            </item>
           </layout>
          </item>
+         <item row="1" column="2">
+          <widget class="QLabel" name="label_connected">
+           <property name="sizePolicy">
+            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
+             <horstretch>0</horstretch>
+             <verstretch>0</verstretch>
+            </sizepolicy>
+           </property>
+           <property name="text">
+            <string>NO</string>
+           </property>
+          </widget>
+         </item>
+         <item row="4" column="0" colspan="2">
+          <widget class="QLabel" name="label_max_rate_text">
+           <property name="text">
+            <string>Max Display Rate</string>
```

Comment:
Let me check if there is something I can do for small screens. It's still important to be able to run this app remotely.

Reply:
This worked fine for me through x-forwarding, fwiw

Reply:
I'm worried that there's something silly like a too-small minimum height on that total section of the display. I'll squint at the ui file for a few minutes.

Reply:
I can reproduce this by going full-screen on my laptop monitor with the camera settings added to the right hand section.
The ui file has no minimum height sizing on any of the right-side elements. I'm not sure if setting them would matter.
This behavior is present in master too (with a sufficiently small monitor).

Since this is also present in master (exasperated here by the addition of new ui elements) I'm going to add this to the jira and handle it in a separate PR.

Reply:
Possibly it will be necessary to add a scrollbar...